pkg/arguments/arguments.go

```diff
@@ -332,15 +335,24 @@ func MarkRegionDeprecated(parentCmd *cobra.Command, childrenCmds []*cobra.Comman
 			deprecateRegion(parentCmd)
 			command.Parent().HelpFunc()(command, strings)
 		})
+
+		disableUsage := "Temporarily used for disabling a warning message ran from other commands (no reason to" +
+			" print for cluster describe called inside cluster create, but there is a use for a lone describe."
+		if cmd.LocalFlags().Lookup(DisableRegionDeprecationFlagName) == nil {
+			cmd.LocalFlags().BoolVar(&DisableRegionDeprecationWarning, DisableRegionDeprecationFlagName,
+				true, disableUsage)
```

Comment:
If the default is true doesn't it always disable it though? Considering you are adding the flag to all commands now :thinking:

Reply:
i dont believe so, it's based on the pointed-to-bool right?

Reply:
oh youre right let me change that just tested

Reply:
fixed